src/main/java/de/adesso/anki/battle/protocol/commands/UseReflectorCommand.java

```diff
@@ -1,27 +1,24 @@
-package com.commands;
-
-import de.adesso.anki.battle.world.DynamicBody;
-import de.adesso.anki.battle.world.World;
-import de.adesso.anki.battle.world.bodies.Vehicle;
-
-public class UseReflectorCommand extends Command {
-
-	public UseReflectorCommand () {
-		
-	}
-	
-	public void execute(Vehicle vehicle) {
-		/*TODO implement  reflection 
-		*/
-
-		World world = vehicle.getWorld();
-		for (DynamicBody dbody : world.getDynamicBodies()) {
-			//reverse  when near
-			double distance = vehicle.getPosition().distance(dbody.getPosition());
-			if (distance< 100){
-				// reverse roadmap
-			}
-		}
-		vehicle.setReflectorReady(false);
-	}
-}
+package de.adesso.anki.battle.protocol.commands;
+
+import de.adesso.anki.battle.world.DynamicBody;
+import de.adesso.anki.battle.world.World;
+import de.adesso.anki.battle.world.bodies.Vehicle;
+
+public class UseReflectorCommand extends Command {
+	
+	@Override
+    public void execute(Vehicle vehicle) {
+		// TODO: implement  reflection
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFf_1B6WA4TguGLC&open=AWKWIFf_1B6WA4TguGLC)

[ISSUE_KEY]: AWKWIFf_1B6WA4TguGLC